### ISSUE TITLE
V-Day Donate banner on moz.org

### ISSUE THREAD

**Issue by kristinashu** [MozillaFoundation/Design#311]
From @WillatMozFdn: moz.org will feature a donate banner from Feb 13-15!

It will be the same as end-of-year #298 and #301 but we should swap out the background image to keep it fresh and Valentines Day-y ❤️ ❤️ ❤️.

cc: @xmatthewx @gideonthomas

**Comment by WillatMozFdn** [comment]
Our friends over on the MoCo side also started a ticket for this one yesterday ... that's here ... https://github.com/mozilla/bedrock/issues/6693 ... does it make sense to combine the two, or just work in one of them?

**Comment by kristinashu** [comment]
This can be the design ticket and https://github.com/mozilla/bedrock/issues/6693 will be the dev implementation ticket. Let's just be sure to add our final design there once it's all approved.

**Comment by gideonthomas** [comment]
@kristinashu do we have copy for this / do you need copy for the design? If so, let's figure out copy in this ticket instead of the bedrock one.

**Comment by WillatMozFdn** [comment]
Hey @gideonthomas ... we're actually (for the first time!) going to do some A/B testing with this one.  Copy still a little tbd, but for now, we can assume the length will be very similar to the EOY banner.  Let me know if you want something that's pretty close to final for generating a comp ... but just know that we may tweak it a little bit, and also have a version B ...

**Comment by gideonthomas** [comment]
Nope. we're good @WillatMozFdn, thanks!

**Comment by kristinashu** [comment]
Hi @WillatMozFdn, here is the updated background design we've been working on https://redpen.io/p/zofc918b58da625a20 

Let me know if you have any feedback/concerns or want to see it with new copy.

**Comment by WillatMozFdn** [comment]
@kristinashu ... this looks great to me, let's go with it.  A question on implementation ... how did we turn it over to the moz.org team for the EOY push?  Did we give them all the code they needed, or did they build substantial chunks of it?  Reason I ask is that we're going to run this A/B test on the verbage this time using traffic cop ... your mockup can be version A (control) ... and version B will be the test, a more "threat-based" version.  Specifically ...

 "Big corporations want to restrict how we access the web. Misinformation is making it harder for us to find the truth. Web-connected devices are pushed to market without meeting minimum security standards. The not-for-profit Mozilla Foundation fights for a healthier internet; **will you donate today?**

Just looking to deliver whatever Eric and Craig need, in a similar format as we did for EOY, so we can establish smooth norms of cooperation for the rest of this year.  Does that make sense?

**Comment by kristinashu** [comment]
Yeah Craig did all the implementation for the EOY banner. We only provided design (background banner image, font styles, and button colors).

I will ask Craig in the implementation ticket https://github.com/mozilla/bedrock/issues/6693 if it is possible to A/B test the banner copy and background image.

I've also added a quick version of what the "threat-based" version could look like: https://redpen.io/a1d3853e-5f42-4b34-c0f7-e43955a75b01
